src/common/src/config.rs

```diff
@@ -825,6 +830,15 @@ impl AsyncStackTraceOption {
     }
 }
 
+#[derive(Debug, Default, Clone, Copy, ValueEnum, Serialize, Deserialize, EnumAsInner)]
+pub enum StrictConsistencyOption {
+    /// Enabled, panic on inconsistency.
+    #[default]
+    On,
+    /// Disabled, warn on inconsistency.
+    Off,
+}
```

Comment:
Isn't it a bool now?

Reply:
I slightly prefer to make it an enum, cuz maybe some day we will introduce other options?